scripts/validate.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 ##===----------------------------------------------------------------------===##
 ##
+## This source file is part of the Hummingbird server framework project
+##
+## Copyright (c) 2021-2024 the Hummingbird authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+##===----------------------------------------------------------------------===##
+##
 ## This source file is part of the SwiftNIO open source project
```

Comment:
Should this text be updated?

Reply:
We need to include the original NIO text, as that is what it was when I copied the file across. I've added a Hummingbird header because the file has been altered since then

Reply:
I see, thanks!